recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set rev = "release_2016.3.2" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "2016.3.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
It was unclear to me glancing at the `setup.py` and src files, but I think this needs to be `numpy x.x` here and in the `run` section.